traffic_incident_parser/launch/traffic_incident_parser.launch

```diff
@@ -16,5 +16,6 @@
 
 <launch>
   <node name="traffic_incident_parser" pkg="traffic_incident_parser" type="traffic_incident_parser_node">
+  <remap from="/environment/geofence" to="incoming_geofence_control"/>
```

Comment:
instead of using /environment the standard in our launch files is to use the environment variables for the different namespaces
$(optenv CARMA_ENV_NS)/
I might have that variable name wrong so check carma_config's carma.env file to confirm it